jupyter/minimal/ubi9-python-3.11/Dockerfile.cpu

```diff
@@ -11,6 +11,15 @@ USER 0
 # Install useful OS packages
 RUN dnf install -y mesa-libGL skopeo && dnf clean all && rm -rf /var/cache/yum
 
+# TexLive dependency for PDF export
+RUN echo "Installing TexLive to allow PDf export from Notebooks" && \
+    wget https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz && \
+    zcat < install-tl-unx.tar.gz | tar xf - && \
+    cd install-tl-2* && \
+    perl ./install-tl --no-interaction --scheme=scheme-small && \
+    cd /usr/local/texlive/2025/bin/x86_64-linux && \
+    ./tlmgr install tcolorbox pdfcol adjustbox titling enumitem soul ucs collection-fontsrecommended
+
```

Comment:
This should go to every JupyterLab-based image we have, including the CUDA and ROCm ones, that have JupyterLab inside.